go/oasis-node/node_test.go

```diff
@@ -72,6 +73,7 @@ var (
 		{workerCommon.CfgClientPort, workerClientPort},
 		{storageWorker.CfgWorkerEnabled, true},
 		{executor.CfgScheduleCheckTxEnabled, false},
+		{tendermintCommon.CfgCoreListenAddress, "tcp://0.0.0.0:27565"},
```

Comment:
Ah yes, I'll put this into a separate commit, basically it is to use a non-default port in case you are also running an actual node on the same address.